build/configs/rtl8721csm/rtl8721csm_download.sh

```diff
@@ -60,11 +61,6 @@ function rtl8721csm_sanity_check()
 		exit 1
 	fi
 
-	if [ ! -f ${TINYARA_BIN} ]; then
```

Comment:
Could you let me know why you remove this?

Reply:
This is because in our platform, the tinyara.bin will not be generated. Checking that file will report error.

Reply:
Oh. TINYARA_BIN definition is missing. The purpose is checking existence of binary which should be downloaded. How about adding the definition as below instead of removing?
```sh
TINYARA_BIN=${BIN_PATH}/km0_km4_image2.bin
```

Reply:
Sure, but it actually has already checked if the km0_km4_image2.bin is missing in line 261
`cp -p ${BIN_PATH}/km0_km4_image2.bin ${IMG_TOOL_PATH}/km0_km4_image2.bin`
If it is missing, it will report error message:
`cp: cannot stat '/root/tizenrt/build/configs/rtl8721csm/../../output/bin/km0_km4_image2.bin': No such file or directory`

I will still add the change you suggested to keep the consistency with other platforms, thanks!

Reply:
Then it's ok. No need to add. Thank you for reply.

Reply:
Noted with thanks!